setup.py

```diff
@@ -1,12 +1,40 @@
-#!/usr/bin/env python
+from setuptools import setup
 
-from distutils.core import setup
-from catkin_pkg.python_setup import generate_distutils_setup
-
-d = generate_distutils_setup(
-    packages=['rqt_graph'],
+package_name = 'rqt_graph'
+setup(
+    name=package_name,
+    version='0.4.10',
     package_dir={'': 'src'},
-    scripts=['scripts/rqt_graph']
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name + '/resource', ['resource/RosGraph.ui']),
+        ('share/' + package_name, ['package.xml']),
+        ('share/' + package_name, ['plugin.xml']),
+        ('lib/' + package_name, ['scripts/rqt_graph'])
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    author='Dirk Thomas',
+    maintainer='Dirk Thomas, Aaron Blasdell',
+    maintainer_email='dthomas@osrfoundation.org',
+    keywords=['ROS'],
+    classifiers=[
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python',
+        'Topic :: Software Development',
+    ],
+    description=(
+        'rqt_topic provides a GUI plugin for displaying debug information about ROS topics '
+        'including publishers, subscribers, publishing rate, and ROS Messages.'
+    ),
+    license='BSD',
```

Comment:
There seems to be a mismatch between this license and the classifiers above.

Reply:
Fixed the classifier